schema.json

```diff
@@ -1046,22 +1046,22 @@
           "type": "object",
           "patternProperties": {
             "propertyNames": {
-              "pattern": "^D[0][2-9]|^D[1-4]D[0-9]"
+              "pattern": "^D[0][2-9]|^D[1-4]D[0-9]^D50"
             },
-            "^D[0][2-9]|^D[1-4]D[0-9]": {
+            "^D[0][2-9]|^D[1-4]D[0-9]^D50": {
```

Comment:
There is an issue with that regex. Should it be `^D[0][2-9]|^D[1-4][0-9]|^D50` instead ?